docs/configuration/configuration.md

```diff
@@ -88,6 +88,7 @@ An example [oauth2-proxy.cfg]({{ site.gitweb }}/contrib/oauth2-proxy.cfg.example
 | `--provider-display-name` | string | Override the provider's name with the given string; used for the sign-in page | (depends on provider) |
 | `--ping-path` | string | the ping endpoint that can be used for basic health checks | `"/ping"` |
 | `--proxy-prefix` | string | the url root path that this proxy should be nested under (e.g. /`<oauth2>/sign_in`) | `"/oauth2"` |
+| `--proxy-time-out` | duration | timeout for upstream proxy | `"30s"` |
```

Comment:
Perhaps we should call this `upstream-request-timeout` so that it is clear that this is the timeout for requests to upstream servers?

Reply:
Changed the name